helm/README.md

```diff
@@ -103,7 +123,75 @@ Example:
 kubectl port-forward -n ${NAMESPACE} nv-ingest-redis-master-0 6379:6379
 ```
 
-### Executing jobs
+#### Executing jobs <= 24.08
+
+Here is a sample invocation of a PDF extraction task using the port forward above:
+
+```bash
+mkdir -p ./processed_docs
+
+nv-ingest-cli \
+  --doc /path/to/your/unique.pdf \
+  --output_directory ./processed_docs \
+  --task='extract:{"document_type": "pdf", "extract_text": true, "extract_images": true, "extract_tables": true}' \
+  --client_host=localhost \
+  --client_port=6379
+```
+
+### Versions > 24.08
+
+Nv-Ingest versions > 24.08 have switched from Redis to a HTTP/Rest based submission method. By default the Rest service runs on port 7670. This means that the `nv-ingest-cli` no longer uses a Redis client so users must use the appropriate version to ensure the client is not still trying to use the RedisClient.
+
+#### Nv-Ingest CLI Installation > 24.08
+
+For versions > 24.08 we recommend building `nv-ingest-cli` from source to ensure you have the latest code. See installation [here](https://github.com/NVIDIA/nv-ingest/tree/main/client)
+
+```bash
+# Just to be cautious we remove any existing installation
+pip uninstall nv-ingest-cli
+
+# Download the 24.08 .whl
+wget https://github.com/NVIDIA/nv-ingest/releases/download/24.08/nv_ingest_client-24.08-py3-none-any.whl
+
+# Pip install that .whl
+pip install nv_ingest_client-24.08-py3-none-any.whl
```

Comment:
Should the versions be `24.09` not 24.08 in these commands?

Reply:
Or, looking at the instructions just above `For versions > 24.08 we recommend building `nv-ingest-cli` from source to ensure you have the latest code`, should it be `pip install /path/to/nv-ingest/client`?